chat_exporter/construct/assets/attachment.py

```diff
@@ -32,13 +32,13 @@ async def build_attachment(self):
 
     async def image(self):
         self.attachments = await fill_out(self.guild, img_attachment, [
-            ("ATTACH_URL", self.attachments.proxy_url, PARSE_MODE_NONE),
-            ("ATTACH_URL_THUMB", self.attachments.proxy_url, PARSE_MODE_NONE)
+            ("ATTACH_URL", self.attachments.url, PARSE_MODE_NONE),
```

Comment:
Using the proxy_url is here intended and better. The proxy url stays a bit longer active when the source was deleted than the url

Reply:
True, I just changed it to use `.url` as its not permanent anymore anyway. However I don't mind changing it back to `.proxy_url`,

Reply:
`proxy_url` is still better compared with `url` - it provides the cached link which assists in some scenarios. Albeit there is an expiration to attachments now.

Reply:
Understood, did a commit to revert those changes.